examples/nlp/language_modeling/megatron_gpt_pretraining.py

```diff
@@ -22,7 +22,7 @@
 from nemo.utils import logging
 from nemo.utils.exp_manager import exp_manager
 
-mp.set_start_method("spawn", force=True)
+# mp.set_start_method("spawn", force=True)
```

Comment:
I'm assuming we're removing this because it causes issues with checkpointing?
what if we start encountering CUDA init issues on NeMo CI?

Reply:
Not removing it, just forgot to uncomment.